microxrce_agent.cpp

```diff
@@ -70,11 +117,113 @@ uint16_t parsePort(const std::string& str_port)
     return valid_port;
 }
 
+void parseMiddleware(
+        const std::vector<std::string>& cl,
+        uint8_t& cl_counter,
+        eprosima::uxr::MiddlewareKind& middleware_kind)
+{
+    if (cl_counter <= cl.size())
```

Comment:
Why not use the cl_it as in the other parameters?.

Reply:
It changes a lot in #78